docs/concepts/browsers-used-by-office-web-add-ins.md

```diff
@@ -12,29 +12,58 @@ Office Add-ins are web applications that are displayed using iFrames when runnin
 > - The [Script Lab tool](../overview/explore-with-script-lab.md) no longer supports Internet Explorer.
 >
 > For more information about supporting Internet Explorer and configuring a graceful failure message on your add-in, see [Support Internet Explorer 11](../develop/support-ie-11.md).
 
-The following table shows which browser is used for the various platforms and operating systems.
+The following sections specify which browser is used for the various platforms and operating systems.
+
+## Non-Windows Platforms
+
+For these platforms, the platform alone determines the browser that is used.
+
+|OS|Office version|Browser|
+|:-----|:-----|:-----|
+|any|Office on the web|The browser in which Office is opened.<br>(But note that Office on the web will not open in Internet Explorer.<br>Attempting to do so opens Office on the web in Edge.) |
+|Mac|any|Safari with WKWebView|
+|iOS|any|Safari with WKWebView|
+|Android|any|Chrome|
+
+## Perpetual Office on Windows
+
+For perpetual Office on Windows, the browser that is used is determined by the operating system, the Office version, whether the license is consumer or commercial, and whether the Edge WebView2 (Chromium-based) is installed.
+
+|OS|Office version|Consumer vs. Commercial|Edge WebView2 (Chromium-based) installed?|Browser|
+|:-----|:-----|:-----|:-----|:-----|
+|Windows 7, 8.1, 10, 11 | Office 2013 | Doesn't matter |Doesn't matter|Internet Explorer 11|
+|Windows 7, 8.1, 10, 11 | Office 2016| Commercial</br>(Build number form is `16.0.xxxx.xxxxx`,</br> ending with two blocks of 4 digits, </br> e.g., 16.0.5197.1000.) |Doesn't matter|Internet Explorer 11|
+|Windows 7, 8.1, 10, 11 | Office 2019| Commercial</br>(Build number form is `1808 (xxxxx.xxxxxx)`,</br> ending with two blocks of 5 digits, </br> e.g., 1808 (Build 10388.20027).)   |Doesn't matter|Internet Explorer 11|
+|Windows 7, 8.1, 10, 11 | Office 2016 to Office 2019| Consumer (Build number form is `YYMM (xxxxx.xxxxxx)`,</br> ending with two blocks of 5 digits, </br> e.g., 2206 (Build 15330.20264).)  |No |Microsoft Edge<sup>1, 2</sup> with original WebView (EdgeHTML)|
+|Windows 7, 8.1, 10, 11 | Office 2016 to Office 2019|  Consumer (Build number form is same as preceding row.)  |Yes<sup>3</sup>|Microsoft Edge<sup>1</sup> with WebView2 (Chromium-based)|
+|Windows 10, 11 | Office 2021| Doesn't matter |Yes<sup>3</sup> |Microsoft Edge<sup>1</sup> with WebView2 (Chromium-based)|
+
+<sup>1</sup> When Microsoft Edge is being used, the Windows Narrator (sometimes called a "screen reader") reads the `<title>` tag in the page that opens in the task pane. When Internet Explorer 11 is being used, the Narrator reads the title bar of the task pane, which comes from the **\<DisplayName\>** value in the add-in's manifest.
```

Comment:
```suggestion
<sup>1</sup> When you use Microsoft Edge, the Windows Narrator (sometimes called a "screen reader") reads the `<title>` tag in the page that opens in the task pane. In Internet Explorer 11, the Narrator reads the title bar of the task pane, which comes from the **\<DisplayName\>** value in the add-in's manifest.
```

Reply:
Is this the behavior the same for JAWS or NVDA? It's estimated that less than 20% of screen reader users use Narrator. JAWS and NVDA combined are about 80%.

Reply:
I'll research that, but it is out-of-scope for this PR.